core/src/main/java/io/questdb/cutlass/http/HttpFullFatServerConfiguration.java

```diff
@@ -59,6 +59,10 @@ default ObjList<String> getContextPathImport() {
         return new ObjList<>("/imp");
     }
 
+    default ObjList<String> getContextPathPreferences() {
+        return new ObjList<>("/preferences");
```

Comment:
this is also quite funky to have "preferences" and "settings", those are synonyms. More standard approach would be to have single URL /settings. To save settings we would send POST /settings and to read GET /settings

Reply:
`/preferences` has been removed.
`/settings` handles GET, POST and PUT.
PUT: overwrite mode, current preferences gets replaced with the one sent in the request
POST: merge mode, current preferences gets merged with the one sent in the request
GET: returns settings which consists of 2 parts, config which comes from server.conf and preferences